.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,66 @@
+name: Publish Draft Release
+# copied from Dafny repo
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*'
+
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  run-tests:
+    name: Rust project - latest
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        toolchain:
+          - stable
+    steps:
+      - name: Checkout Cedar Examples
+        uses: actions/checkout@v3
```

Comment:
what does this do

Reply:
It checks out cedar-java. Fixed the comment here and in publish_release_reusable.yml